src/sentry/services/hybrid_cloud/organization.py

```diff
@@ -12,36 +25,109 @@
 from sentry.silo import SiloMode
 
 
+@dataclass
+class ApiTeam:
+    id: int = -1
+    status: int = TeamStatus.VISIBLE
```

Comment:
can you not type check against an enum?

Reply:
So unfortunately `TeamStatus` isn't actually an enum -- there's a comment to that effect.  It's just an int.  That was part of the motivating reason for me to create an ApiTeamStatus enum, but it was also redundant.  Ideally we'd change the type of the existing enum, but that becomes a somewhat annoying PR change due to fixing a few other callsites.  We can consider doing it in another pass thought.